### PR TITLE
feat: elab: deadline, new testamonial, remove section

### PR DESCRIPTION
**Pull request by MunzerDw**

### CODE REVIEW

File: pages/e-lab.tsx [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
   faChevronDown,
   faCircleNodes,
   faHandshakeSimple,
   faHandsHoldingCircle,
 } from "@fortawesome/free-solid-svg-icons";
-import Benefits from "@components/Benefit";
-import Timeline from "@components/Timeline";
-import FAQ from "@components/FAQ";
-import { faq } from "data/e-lab";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (simon-hng) [inline pages/e-lab.tsx:16]: why

Comment (MunzerDw) [reply pages/e-lab.tsx:16]: I swear I didnt do it

Comment (MunzerDw) [reply pages/e-lab.tsx:16]: It was there before, my linter just moved the imports order

Comment (simon-hng) [reply pages/e-lab.tsx:16]: oh okay. But why does your linter do that?

File: data/e-lab.tsx [hunk 2]
```diff
@@ -75,6 +75,13 @@ export const testimonials = [
     logoSrc: "/assets/e-lab/partners/uvc_w.svg",
     logoAlt: "UVC Partners",
   },
+  {
+    imgSrc: "/assets/e-lab/testimonials/maximilian_jesch.png",
+    name: "Maximilian Jesch (Jury)",
+    text: "Being a jury member at the AI Startup Contest was an exhilarating experience! The passion and innovation displayed by the participating teams left me truly impressed. Their groundbreaking ideas and impeccably delivered pitches showcased the immense potential of the next generation of AI-driven entrepreneurship. I have no doubt that these teams are on the cusp of revolutionizing the industry and driving us into an exciting AI-powered future. It's an honor to be part of their journey, and I eagerly anticipate witnessing their continued success and impact on the world.",
+    logoSrc: "",
```
Comment (simon-hng) [inline data/e-lab.tsx:82]: Missing logo? I didn't quite follow the convo on slack

File: pages/e-lab.tsx [hunk 3]
```diff
@@ -359,8 +356,9 @@ export default function AIELab() {
 
         <div className="flex flex-col items-center justify-center gap-8">
           <p>
-            Do you want to become part of a commmunity of AI enthusiasts and
-            found your AI startup? We are excited to have you on board.
+            Do you want to become part of a community of AI enthusiasts and
+            found your AI startup? Don’t miss your chance — applications close
```
Comment (simon-hng) [inline pages/e-lab.tsx:360]: do we need `&apos;` here?

**Review by simon-hng — APPROVED** [review]

File: data/e-lab.tsx (hunk 2 above)
Comment (timonschramm) [reply data/e-lab.tsx:82]: IBM, did not gave permission to use theirs, so no :(

File: pages/e-lab.tsx (hunk 3 above)
Comment (timonschramm) [reply pages/e-lab.tsx:360]: Agree, for me it caused problems in the past, when I did not use them.

**Pull request merged**